src/app/project/ontology/resource-class-form/resource-class-form.component.ts

```diff
@@ -344,9 +367,58 @@ export class ResourceClassFormComponent implements OnInit, OnDestroy, AfterViewC
     submitData() {
         this.loading = true;
         if (this.edit) {
-            // edit mode
-            // submit properties and set cardinality
-            this.submitProps(this.resourceClassForm.value.properties, this.iri);
+
+            if (this.showResourceClassForm) {
+                // edit mode: res class info (label and comment)
+                // label
+                const onto4Label = new UpdateOntology<UpdateResourceClassLabel>();
```

Comment:
I'm not sure I understand this variable name 😅 same with `onto4comment`

Reply:
I can change it. I think I copied from the js-lib example 🙈

Reply:
It's the `UpdateOntology` object. And we need an `UpdateOntology` object for the `label` and one for the `comment`. So, it's an ontology for label (`onto4label`) and an ontology for comment (`onto4comment`)